_docs/index.html

```diff
@@ -43,7 +43,7 @@ <h2 class="title">Add Yourself to the Teams Page</h2>
     <div class="grid-row">
       <div class="grid-col">
           <div class="profile-img-container">
-                <img class="profile-img" src="https://api.adorable.io/avatars/512/ucla.png" alt="profile picture"/>
+                <img class="profile-img" src="../img/blank-profile.jpg" alt="profile picture"/>
```

Comment:
One quick note here - this is something that we haven't talked about yet, but we typically don't want to use `../` or relative paths since the final output path can be easily changed. Instead, we'll use a full path from the site root, using Jekyll's `site.baseurl` property (I'll explain this more-indepth either in OH or in a future meeting).

Long story short, the change should be:

```
<img class="profile-img" src="{{site.baseurl}}/img/blank-profile.jpg" alt="sample profile picture"/>
```

Here, the `{{}}` is telling liquid to evaluate and output the liquid expression, which is the config property `site.baseurl`. 

I also added a bit more information in the `alt` tag.